webapp/app/templates/components.html

```diff
@@ -52,23 +52,35 @@
 {% endmacro %}
 
 {% macro radio_field(field) -%}
-    {% for subfield in field %}
-        <div class="form-row-center">
-            {{ subfield(class="") }}
-            {{ field_label(subfield, class="col-sm-10 col-form-label") }}
-        </div>
-    {% endfor %}
+    <fieldset>
+        {{ field_label(field) }}
+        {% for subfield in field %}
+            <div>
+                {{ subfield(class="") }}
+                {{ field_label(subfield, class="col-sm-10 col-form-label") }}
+            </div>
+        {% endfor %}
+        {{ field_errors(field) }}
+    </fieldset>
```

Comment:
We could think about putting that part in a macro. But this is probably only a good idea if we have to use the fieldset more often.

Reply:
Yes, I think so too.